rust/src/rfb/rfb.rs

```diff
@@ -18,15 +18,20 @@
 // Author: Frank Honza <frank.honza@dcso.de>
 
 use std;
-use std::ffi::CString;
-use crate::core::{ALPROTO_UNKNOWN, AppProto, Flow, IPPROTO_TCP};
```

Comment:
It's because you removed this import in this commit and added it back in the next commit. So, the commit should ideally fail to build.

Reply:
Please make sure that the fmt commit only does the formatting and nothing else.